substra/sdk/client.py

```diff
@@ -97,24 +95,29 @@ def __init__(self, config_path=None, profile_name=None, user_path=None,
         # set current logged user if exists
         self.set_user()
 
-    @logit
-    def login(self):
-        """Login.
-
-        Allow to login to a remote server.
+        if token:
+            self._set_token(token)
 
-        After setting your configuration with `substra config` using `-u` and `-p`
-        Launch `substra login`
-        You will get a token which will be stored by default in `~/.substra-user`
-        You can change that thanks to the --user option (works like the --profile option)
+    @logit
+    def login(self, username, password):
+        """Login to a remote server.
 
+        Will log into the current profile's server and save the resulting token in the configured
+        user_path.
```

Comment:
At first sight it looks like the resulting token is not stored in the user_path. Correct me if I'm wrong, I think this is valid only for the CLI login command.

Reply:
Yes you're right, I'll update the docstring.